tests/unit_tests/test_other.py

```diff
@@ -248,6 +248,170 @@ def test_keyreg_txn_nonpart(self):
             encoding.msgpack_encode(encoding.future_msgpack_decode(keyregtxn)),
         )
 
+    def test_stateproof_txn(self):
```

Comment:
Is this one of the msgpack tests that the python cucumber test doesn't support?

Reply:
yes, I extracted the txn object from v2_indexer_lookupBlock_transaction.json from encoded it using msgpacktool.